### PR TITLE
inv: Add LoongArch64 machines

### PR DESCRIPTION
**Pull request by Panxuefeng-loongson**
According to add a new machine manual: https://github.com/adoptium/infrastructure/wiki/Adding-a-new-machine, I'm submitting this PR for advice.

The information on this machine is as follows:
```
Architecture:        loongarch64
Byte Order:          Little Endian
CPU(s):              4
On-line CPU(s) list: 0-3
Thread(s) per core:  1
Core(s) per socket:  4
Socket(s):           1
NUMA node(s):        1
Model name:          Loongson-3A5000
CPU max MHz:         2500.0000
CPU min MHz:         225.0000
BogoMIPS:            3194.88
L1d cache:           64K
L1i cache:           64K
L2 cache:            256K
L3 cache:            16384K
NUMA node0 CPU(s):   0-3
```

### CODE REVIEW

**Comment by Panxuefeng-loongson** [comment]
@sxa @gdams @karianna I would like you give me some advice on this PR，thanks

**Review by karianna — APPROVED** [review]
LGTM but over to @sxa and team

**Comment by sxa** [comment]
@tellison ?

**Comment by Panxuefeng-loongson** [comment]
I have communicated with tim via email and also created a issue: https://github.com/adoptium/infrastructure/issues/2836. Tim has left a comment in this issue: https://github.com/adoptium/infrastructure/issues/2836#issuecomment-1331843852. @sxa Do you have any comments on this? Thanks

File: ansible/inventory.yml [hunk 1]
```diff
@@ -89,6 +89,9 @@ hosts:
       - spearhead:
           freebsd12-x64-1: {ip: 185.131.222.224}
 
+      - loongson:
+          Loongnix20-loongarch64-1: {ip: 10.40.64.29, user: loongson}
```
Comment (theaoqi) [inline ansible/inventory.yml:93]: I see the style of os name is lowercase. How about `loongnix20-loongarch64-1`?

Comment (Panxuefeng-loongson) [reply ansible/inventory.yml]: Sorry, my fault.

**Comment by sxa** [comment]
> I have communicated with tim via email and also created a issue: #2836. Tim has left a comment in this issue: [#2836 (comment)](https://github.com/adoptium/infrastructure/issues/2836#issuecomment-1331843852). @sxa Do you have any comments on this? Thanks

My personal feeling is that while I'm ok to have PRs into the playbooks to support people configuring their machines for Loongson and also to accept PRs into the build repositories for adding them to our build scripts, I'm not sure we can justify adding it into our CI at this point, especially since Temurin is not currently planning to release on that platform.

**Comment by Panxuefeng-loongson** [comment]
Hi @sxa , Thank you for your reply. I'm sorry I didn't make it clear to you what loongson is trying to achieve by joining the community. I think I can paraphrase the email that our team leader @theaoqi replied to tim, it may not be polite, but the message in the email makes it clear what our goal is:

(1) The simplest level is adding the  build and test code to support our development work, Loongson uses Adoptium projects code to improve the quality of our JDK.
(2) We want to  run build and test on Adoptium infrastructure, for example nightly/weekly tests. This enables us to compare tests with other builders, and ensure that the code we have enabled works well. As far as my current understanding, to achieve this level, adoptium's infrastructure may need to have LoongArch machines and some pipeline scripts need to be modified, so I want to push this thing.

The above two steps are the goals we expect to achieve, perhaps our company has more in-depth plans, but I can't promise at present. The simplest level seems to be almost reached, now I want to reach the second step of the goal. The prerequisites for the second step may not be met, @sxa You can make suggestions and then we meet the Adoptium's requirements.

@theaoq is our team leader, You can also contact him(Email: aoqi@loongson.cn) and he will answer your questions.

**Comment by sxa** [comment]
> We want to run build and test on Adoptium infrastructure, for example nightly/weekly tests.

Thanks - will aim to sync with Tim again today since I haven't been on the email threads.

**Comment by Panxuefeng-loongson** [comment]
> > I think I can paraphrase the email that our team leader @theaoqi replied to tim, it may not be polite, but the message in the email makes it clear what our goal is:
> > We want to run build and test on Adoptium infrastructure, for example nightly/weekly tests.
> 
> Thanks - will aim to sync with Tim again today since I haven't been on the email threads.

I added you to the cc list when I sent an email to tim two days ago. I don't know if you got enough information from the email, but if you still have questions, please contact me and I will help you.

**Comment by sxa** [comment]
> > > I think I can paraphrase the email that our team leader @theaoqi replied to tim, it may not be polite, but the message in the email makes it clear what our goal is:
> > > We want to run build and test on Adoptium infrastructure, for example nightly/weekly tests.
> > Thanks - will aim to sync with Tim again today since I haven't been on the email threads.
> I added you to the cc list when I sent an email to tim two days ago. I don't know if you got enough information from the email, but if you still have questions, please contact me and I will help you.

Thanks. I think the key phrase of Tim's response last week is _"With project approval..._" and we are having those discussions with Eclipse at the moment to see if it's something we can accommodate.

**Comment by Panxuefeng-loongson** [comment]
> > > > I think I can paraphrase the email that our team leader @theaoqi replied to tim, it may not be polite, but the message in the email makes it clear what our goal is:
> > > > We want to run build and test on Adoptium infrastructure, for example nightly/weekly tests.
> 
> > > Thanks - will aim to sync with Tim again today since I haven't been on the email threads.
> 
> > I added you to the cc list when I sent an email to tim two days ago. I don't know if you got enough information from the email, but if you still have questions, please contact me and I will help you.
> 
> Thanks. I think the key phrase of Tim's response last week is _"With project approval..._" and we are having those discussions with Eclipse at the moment to see if it's something we can accommodate.

Thanks. After the committee discussion, if there are any issues that we still need to address, please let us know and we will consider them as the highest priority.

**Comment by sxa** [comment]
Based on an discussion with @tellison earlier today it sounds like we are not in a position to support this in our CI until such time as the loongarch port is merged into the upstream openjdk codebase. Once this is in place we can consider adding it.

**Pull request closed without merging**